trie-db/src/lookup.rs

```diff
@@ -167,7 +179,13 @@ where
 				full_key,
 				|v, _, full_key, _, recorder, _| {
 					Ok(match v {
-						Value::Inline(v) => L::Hash::hash(&v),
+						Value::Inline(v) => {
+							if let Some(recoder) = recorder.as_mut() {
+								recoder.record(TrieAccess::InlineValue { full_key });
```

Comment:
Maybe a comment to indicate we prefer recording a value access than a hash here.